commit 47739c96de94d8de07ee1e8d92cf845d083cb422
Author: Bob Peterson <rpeterso@redhat.com>
Date:   Tue May 14 15:00:22 2013 -0500

    fsck.gfs2: Remove unused variable
    
    This patch removes a variable that wasn't being used.
    
    rhbz#902920

diff --git a/gfs2/fsck/pass1b.c b/gfs2/fsck/pass1b.c
index c7889f6..bf26c2c 100644
--- a/gfs2/fsck/pass1b.c
+++ b/gfs2/fsck/pass1b.c
@@ -144,7 +144,6 @@ static void resolve_dup_references(struct gfs2_sbd *sdp, struct duptree *dt,
 	enum dup_ref_type this_ref;
 	struct inode_info *ii;
 	int found_good_ref = 0;
-	uint64_t dup_block;
 	uint8_t q;
 
 	osi_list_foreach_safe(tmp, ref_list, x) {
@@ -226,7 +225,6 @@ static void resolve_dup_references(struct gfs2_sbd *sdp, struct duptree *dt,
 				 (unsigned long long)id->block_no,
 				 (unsigned long long)id->block_no);
 
-		dup_block = id->block_no;
 		ip = fsck_load_inode(sdp, id->block_no);
 		/* If we've already deleted this dinode, don't try to delete
 		   it again. That could free blocks that used to be duplicate
